README.md

```diff
@@ -59,7 +59,7 @@ The following tools help make easier to work with sample code.
 
 ## Authors
 
-👤 **Benjamin**
+👤 **Benjamin** 
 
 - GitHub: [@benjp009](https://github.com/benjp009)
 
```

Comment:
- [ ] Please kindly delete the `node_modules` directory from the repository and add it to the `.gitignore` file as it is not necessary and it make your repository to be crowed.